package.json

```diff
@@ -28,22 +28,29 @@
   "dependencies": {
     "@google-cloud/common": "^0.26.0",
     "@types/console-log-level": "^1.4.0",
+    "@types/lodash": "^4.14.115",
```

Comment:
It seems that these `@types` dependencies have sneaked in here. Should be in `devDependencies`.

Reply:
Done. 

Some of these may have to be added back to be able to the module with typescript, though (the e2e test will fail if this is the case)

Reply:
the `@types/` dependencies that I've kept are necessary for typescript complication.

Reply:
`@types/` dependencies should never be in `dependencies`. Anything in your dependencies will be deployed to users' production. I left some more details thoughts on this topic [elsewhere](https://github.com/grpc/grpc-node/pull/615#discussion_r233265995).

This discussion can probably be moved to a new issue, but I would like to see what issues each of `@types/semver` and `@types/console-log-level` cause. Can you include the e2e compile error message in those cases in a new issue?